pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -199,6 +199,15 @@ type SchedulerVolumeBinder interface {
 	BindPodVolumes(ctx context.Context, assumedPod *v1.Pod, podVolumes *PodVolumes) error
 }
 
+// PodVolumesSeparated stores a pod's PVCs separated into bound, unbound with delayed binding (including provisioning),
+// unbound with immediate binding (including prebound) and pvs that belong to storage classes of unbound PVCs with delayed binding.
+type PodVolumesSeparated struct {
```

Comment:
actually we can call it `PodVolumeClaims`, that is what it includes for the most part. We can also change the function name to reflect that, `GetPodVolumeClaims`